ci-shell.nix

```diff
@@ -7,5 +7,6 @@ pkgs.mkShell {
     firefox-esr-bin
     firefox-nightly-bin
     librewolf
+    floorp-darwin
```

Comment:
I'd rename this to `floorp-bin` for the sake of naming consistency

Reply:
Sounds good